### PR TITLE
Update README on how to add custom claims to JWT access token

### PR DESCRIPTION
**Pull request by arvindkrishnakumar-okta**
### Issue:

[OKTA-323354](https://oktainc.atlassian.net/browse/OKTA-323354)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -209,6 +209,41 @@ static class WebConfig extends WebSecurityConfigurerAdapter {
 }
 ```
```
Comment (arvindkrishnakumar-okta) [inline README.md:211]: @bdemers Should we reference https://developer.okta.com/docs/guides/customize-tokens-returned-from-okta/add-custom-claim/ here?

File: README.md [hunk 2]
```diff
@@ -228,6 +228,8 @@ public class CustomTokenEnhancer implements TokenEnhancer {
 Now, add it to the Authorization Server configuration:
  
 ```java
+@Configuration
+@EnableAuthorizationServer
```
Comment (bdemers) [inline README.md:232]: This is the wrong annotation (it's also deprecated)

File: README.md (hunk 1 above)
Comment (bdemers) [reply README.md:211]: I think I'm missing something here.  You are creating an Authorization server in the example?

Comment (arvindkrishnakumar-okta) [reply README.md:211]: Nope, I am not creating an AS in the example.

File: README.md (hunk 2 above)
Comment (arvindkrishnakumar-okta) [reply README.md]: removed

File: README.md [hunk 3]
```diff
@@ -209,6 +209,23 @@ static class WebConfig extends WebSecurityConfigurerAdapter {
...
+If you want to add custom claims to JWT tokens in your custom Authorization Server, see [Add Custom claim to a token](https://developer.okta.com/docs/guides/customize-tokens-returned-from-okta/add-custom-claim/) for more info.
+
+You could then extract the attributes from the token by doing something like below:
+
+```java
+@RestController
+public class ExampleController {
+
+    @GetMapping("/email")
+    @PreAuthorize("hasAuthority('SCOPE_profile')")
```
Comment (bdemers) [inline README.md:221]:
```suggestion
```

File: README.md [hunk 4]
```diff
@@ -209,6 +209,38 @@ static class WebConfig extends WebSecurityConfigurerAdapter {
...
+```
+
+**Resource Server Flow:**
+
+```java
+@RestController
+public class MessageOfTheDayController {
+
+    @GetMapping("/api/userProfile")
+    @PreAuthorize("hasAuthority('SCOPE_profile')")
```
[the remark this answers is not in the document]
Comment (arvindkrishnakumar-okta) [reply README.md]: already done.

**Review by bdemers — APPROVED** [review]
My fault for missing this one :D

**Comment by arvindkrishnakumar-okta** [comment]
> My fault for missing this one :D

no problem :)

**Pull request merged**